Makefile.devel

```diff
@@ -334,26 +318,25 @@ tag:
 		rm $$f.old; \
 	   done) \
 	fi
-	if [ -z "$(NOCVS)" ]; then cvs commit -m"Set version tag for new release." generic/$(VERSIONFILE) etc/ProofGeneral.spec; (cd $(HTMLDIR); cvs commit -m"Set version tag for new release." $(DOWNLOADHTMLS)); fi
-	if [ -z "$(NOCVS)" ]; then cvs tag -cF "$(CVS_RELEASENAME)"; fi
+	if [ -z "$(NOCVS)" ]; then git add generic/$(VERSIONFILE) etc/ProofGeneral.spec && git commit -m"Set version tag for new release." ; git add $(HTMLDIR)/$(DOWNLOADHTMLS) && git commit -m"Set version tag for new release."; fi
+	if [ -z "$(NOCVS)" ]; then git tag -f  "$(GIT_RELEASENAME)"; fi
```

Comment:
You should replace `git tag -f  "$(GIT_RELEASENAME)"` with `git tag -a "$(GIT_RELEASENAME)"`